composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": ">=5.6",
+        "behat/mink-selenium2-driver": "dev-master as 1.3.x-dev",
```

Comment:
I believe this should be in the "require-dev" section.

Reply:
I can move it, but from what I see in https://github.com/drupal-composer/drupal-project/issues/511#issuecomment-527382980 has it in the `require` section.

Still move it?

Reply:
Yes I think so since it is a dev dependency in the first place.

https://github.com/minkphp/MinkSelenium2Driver/issues/313#issuecomment-527148679

I used "1.4.x-dev as 1.3.x-dev" locally to be able to work on a project.

Reply:
This PR shouldn't be required now - https://packagist.org/packages/behat/mink-selenium2-driver#1.3.x-dev